Marlin/src/gcode/motion/G2_G3.cpp

```diff
@@ -290,7 +290,7 @@ void GcodeSuite::G2_G3(const bool clockwise) {
                       len = d2.magnitude(),              // Distance to mid-point of move from current
                       h2 = (r - len) * (r + len),        // factored to reduce rounding error
                       h = (h2 >= 0) ? SQRT(h2) : 0.0f;   // Distance to the arc pivot-point from midpoint
-          const xy_pos_t s = { -d2.y, d2.x } / len;      // Unit vector along perpendicular bisector
+          const xy_pos_t s = { -d2.y/len, d2.x/len};     // Unit vector along perpendicular bisector
```

Comment:
If the division is failing, let's look at fixing `XYVar` which is supposed to have no issue with this.

Reply:
Aha. Of course. This works: `xy_pos_t({ -d2.y, d2.x }) / len`.

Reply:
We'll just move the division down to the next line….